docs/sources/http_api/data_source.md

```diff
@@ -226,7 +226,7 @@ Content-Type: application/json
 }
 ```
 
-> NOTE: `password` and `basicAuthPassword` should be defined under `secureJsonData` in order to be stored securely as an encrypted blob in the database. Then, the encrypted fields are listed under `secureJsonFields` section in the response. See also the [Encrypting Sensitive Data]({{< relref "../developers/plugins/authentication.md/#encrypting-sensitive-data">}}) documentation for more details.
+> NOTE: `password` and `basicAuthPassword` should be defined under `secureJsonData` in order to be stored securely as an encrypted blob in the database. Then, the encrypted fields are listed under `secureJsonFields` section in the response.
```

Comment:
```suggestion
> **Note:** Define `password` and `basicAuthPassword` should be defined under `secureJsonData` so that they are stored securely as an encrypted blob in the database. The encrypted fields are listed under `secureJsonFields` section in the response.
```

Reply:
Ugh, this note is awful. So much passive. I don't have a fix for it right now, though.

Reply:
```suggestion
> **Note:** By defining `password` and `basicAuthPassword` under `secureJsonData` Grafana encrypts them securely as an encrypted blob in the database. The response then lists the encrypted fields under `secureJsonFields`.
```

Better?